code/modules/mob/living/silicon/decoy/decoy.dm

```diff
@@ -37,7 +37,8 @@
 	if(stat == DEAD)
 		return FALSE
 	icon_state = "hydra-off"
-	addtimer(CALLBACK(GLOBAL_PROC, GLOBAL_PROC_REF(explosion), loc, -1, 0, 8, 12), 2 SECONDS)
+	var/datum/cause_data/cause_data = create_cause_data("rapid unscheduled disassembly", src, src)
+	addtimer(CALLBACK(GLOBAL_PROC, GLOBAL_PROC_REF(explosion), loc, -1, 0, 8, 12, , , , cause_data), 2 SECONDS)
```

Comment:
```suggestion
	addtimer(CALLBACK(GLOBAL_PROC, GLOBAL_PROC_REF(explosion), loc, -1, 0, 8, 12, TRUE, FALSE, 0, cause_data), 2 SECONDS)
```
just pass the default values through instead (despite these not being used at all, it's just ugly otherwise)